gov_uk_dashboards/components/plotly/phase_banner.py

```diff
@@ -3,11 +3,15 @@
 
 
 def phase_banner_with_feedback(
```

Comment:
can we update the version number in setup.py - I think patch.

Reply:
Thanks, I couldn't remember which file to do this in!